Cargo.toml

```diff
@@ -20,4 +20,6 @@ include = [
 
 [dev-dependencies]
 lazy_static = "1.0"
+maplit = "1"
+pretty_assertions = "0.5"
```

Comment:
Is it common practice to leave out "trailing" zeros? It would seem more consistent to provide the full crate IDs.